flow360/component/geometry.py

```diff
@@ -231,9 +201,9 @@ def submit(self, progress_callback=None) -> Geometry:
         submitted_geometry = Geometry(self.id)
 
         remote_file_names = []
-        for index, geometry_file in enumerate(self.file_names):
+        for geometry_file in self.file_names:
             _, ext = os.path.splitext(geometry_file)
-            remote_file_name = f"geometry_{index}{ext}"
+            remote_file_name = f"geometry{ext}"
```

Comment:
are you sure we want to rename files on upload? This will break assemblies

Reply:
fixed. The geometry file won't be renamed to support assembly files